pkg/hydrator/nexpose.go

```diff
@@ -171,13 +171,27 @@ type NexposeClient struct {
 	PageSize   int
 }
 
+// PermanentError is used as a signal for callers of makeNexposeRequest
+// to handle permanent errors from Nexpose
+type PermanentError struct {
+}
+
+func (pe *PermanentError) Error() string {
+	return "Permanent Nexpose Error"
```

Comment:
Suggestion: I think "Permanent Nexpose Error" is confusing here. Nexpose shouldn't be sending us back these errors and they're not intended to be permanent errors. Maybe more comments and some wording that conveys what's actually happening here would help.